installing/installing-troubleshooting.adoc

```diff
@@ -1,5 +1,5 @@
-[id="installing-gather-logs"]
-= Gathering installation logs
+[id="installing-troubleshooting"]
+= Troubleshooting installation issues
```

Comment:
Re-titling to reflect our customers' intentions and the increasing scope of the content.